accord-core/src/main/java/accord/local/SaveStatus.java

```diff
@@ -259,6 +275,7 @@ public static SaveStatus enrich(SaveStatus status, Known known)
         return status;
     }
 
+    // TODO (expected): tighten up distinction of "preferKnowledge" and its interaction with CheckStatus
```

Comment:
I don't think it can, logically. Not without super complicating other parts of the state machine anyway. The right thing to do is to move more fully to `Known`, probably - since this reifies the distinctions we care about. Hopefully the recent refactor brings us close enough for now, but I haven't removed these registers yet.